test/index.js

```diff
@@ -1,11 +1,7 @@
-import './assignment-tests';
-import './compose-basic-tests';
-import './compose-tests';
-import './descriptor-tests';
-import './initializer-tests';
-import './merge-tests';
-import './priority-tests';
-import './property-descriptor-tests';
-import './stamp-tests';
-import './instance-replacement-tests';
-import './property-safety-tests.js';
+require('babel-register');
```

Comment:
The code you proposed won't work as the `./run-all` file expects the `global.compose` variable to exist. The following won't work either because all the `import` statements are executed the first. The JS code itself is executed after. Thus, we have to use `require`.
```js
import compose from '../examples/compose';
global.compose = compose;
import './run-all';
```